testcases/contenttype.http

```diff
@@ -0,0 +1,11 @@
+POST / HTTP/1.1
+Host: handler.example.com
+Content-Type: application/xml
+Content-Length: 24
+ce-specversion: 0.4-wip
+ce-id: 123
+ce-source: https://cloudevents.io/contenttype
+ce-type: io.cloudevents.contenttype-test
+ce-datacontentencoding: Base64
```

Comment:
should this be `Content-Encoding`?  the normal http header

I think we need to update the http spec to say this if so

Reply:
#488 for this, and to clarify percent-encoding to be a bit more coherent.

Reply:
I didn't apply #488 here yet, and the rules in #488 right now would also require percent-encoding "/" and ":" in URLs.

Reply:
@evankanderson can you update hits per the latest stuff in the spec?